xdsl/interactive/app.py

```diff
@@ -159,10 +159,11 @@ class InputApp(App[None]):
     """
 
     pre_loaded_input_text: str
-    pre_loaded_pass_pipeline: tuple[tuple[type[ModulePass], PipelinePassSpec], ...]
+    pre_loaded_file_path: str
```

Comment:
I would say that when the user selects a file from within the app, this should also be updated, so this should be more like `file_path: str | None`, and we should have appropriate logic to set it to None when the user modifies the text in the input, until we have a way to save the file

Reply:
Actually let's skip the logic for setting it to None, but it would be good to update the file name in the gui